freqtrade/optimize/optimize_reports.py

```diff
@@ -66,7 +66,7 @@ def generate_text_table(data: Dict[str, Dict], stake_currency: str, max_open_tra
     ])
     # Ignore type as floatfmt does allow tuples but mypy does not know that
     return tabulate(tabular_data, headers=headers,
-                    floatfmt=floatfmt, tablefmt="pipe")  # type: ignore
+                    floatfmt=floatfmt, tablefmt="orgtbl", stralign="right")  # type: ignore
```

Comment:
why is it `"orgtbl"` here - and `"psql"` in `list_commands.py`? 

is it because of the closing (lower border) of the table ?

Reply:
What i found is that "orgtbl" is the same as "psql" but without the top en bottom border.
In order to keep the style of backtesting results i used orgtbl.

So there are two options with "same style" but with and without top/bottom bar.

Example "psql":
```
+-------------+--------------+---------+---------+----------+
|          Id |       Symbol |    Base |   Quote |   Active |
|-------------+--------------+---------+---------+----------|
|      ADABNB |      ADA/BNB |     ADA |     BNB |     True |
|      ADABTC |      ADA/BTC |     ADA |     BTC |     True |
|     ADABUSD |     ADA/BUSD |     ADA |    BUSD |     True |
|      ADAETH |      ADA/ETH |     ADA |     ETH |     True |
+-------------+--------------+---------+---------+----------+
```
Example "orgtbl":
```
|          Id |       Symbol |    Base |   Quote |   Active |
|-------------+--------------+---------+---------+----------|
|      ADABNB |      ADA/BNB |     ADA |     BNB |     True |
|      ADABTC |      ADA/BTC |     ADA |     BTC |     True |
|     ADABUSD |     ADA/BUSD |     ADA |    BUSD |     True |
|      ADAETH |      ADA/ETH |     ADA |     ETH |     True |
```

Reply:
that's fine - just confirming :+1: